extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/render/ImageRenderer.java

```diff
@@ -54,7 +56,9 @@ public boolean canRender(Widget w) {
 
         String sitemap = w.eResource().getURI().path();
 
-        String url = "../proxy?sitemap=" + sitemap + "&amp;widgetId=" + widgetId + "&amp;t=" + (new Date()).getTime();
+        State state = itemUIRegistry.getState(w);
+        String url = (state instanceof RawType) ? state.toFullString()
+                : ("../proxy?sitemap=" + sitemap + "&amp;widgetId=" + widgetId + "&amp;t=" + (new Date()).getTime());
```

Comment:
While this works for now, further support will also require adding classes to image element so it can be handled in javascript